### PR TITLE
Update adding-analytics.md

### PR DESCRIPTION
**Pull request by baumant**

### CODE REVIEW

File: docs/docs/adding-analytics.md [hunk 1]
```diff
@@ -2,7 +2,57 @@
...
+You can find this tracking ID later by going to `Admin > Tracking Info > Tracking Code`.
+
+Now to configure Gatsby to send page views to your Google Analytics account.
+
+We are going to use `gatsby-plugin-google-analytics`
+
+```npm install --save gatsby-plugin-google-analytics```
+
+In your gatsby-config.js:
+```jsx
```
Comment (LekoArts) [inline docs/docs/adding-analytics.md]: You can add the filename in the code block (then you can remove the "in your gatsby-config.js" statement)

```suggestion
```jsx:title=gatsby-config.js
```

File: docs/docs/adding-analytics.md [hunk 2]
```diff
@@ -2,7 +2,57 @@
...
+
+Once you have an account you will be prompted to set up a new property. This property will have a Tracking ID associated with it. In this case the property will be the website itself. Fill out the below form with your website name and URL
+
+Once complete you should now have a Tracking ID take note of it as your website will need to reference it when sending page views to Google Analytics. It should be in the format `UA-XXXXXXXXX-X`.
+
+You can find this tracking ID later by going to `Admin > Tracking Info > Tracking Code`.
+
+Now to configure Gatsby to send page views to your Google Analytics account.
+
+We are going to use `gatsby-plugin-google-analytics`
```
Comment (LekoArts) [inline docs/docs/adding-analytics.md]: Maybe a comment on what other Google plugins exist (gtag, tag manager)?

File: docs/docs/adding-analytics.md [hunk 3]
```diff
@@ -2,7 +2,57 @@
...
+Here replace `UA-XXXXXXXXX-X` with your own Tracking ID.
+
+Full documentation for the plugin can be found in the Gatsby plugins documentation [here](https://www.gatsbyjs.org/packages/gatsby-plugin-google-analytics/) or on the Github page [here](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-google-analytics)
+
+In the example we have used `head: true` as [suggested in Google's documentation](https://developers.google.com/analytics/devguides/collection/analyticsjs/). This will put the tracking script in the head. This defaults to false which will instead put it in the body. There are a number of extra configuration options both with the Gatsby plugin and also on Google Analytics itself so you can tailor to meet your website's needs.
+
+Once this is configured you can deploy your site to test! If you navigate to Home on Google Analytics you should see a dashboard with different statistics.
+
+### Other Gatsby analytics plugins
+- [Segment](https://www.gatsbyjs.org/packages/gatsby-plugin-segment)
```
Comment (DSchau) [inline docs/docs/adding-analytics.md]: These should be relative links, e.g. `/packages/whatever`

File: docs/docs/adding-analytics.md (hunk 1 above)
Comment (baumant) [reply docs/docs/adding-analytics.md]: fixed, changed to just js as well

File: docs/docs/adding-analytics.md (hunk 2 above)
Comment (baumant) [reply docs/docs/adding-analytics.md]: added anchor link to other analytics options

File: docs/docs/adding-analytics.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (baumant) [reply docs/docs/adding-analytics.md]: fixed

File: docs/docs/adding-analytics.md [hunk 4]
```diff
@@ -2,7 +2,57 @@
...
+
+Once complete you should now have a Tracking ID take note of it as your website will need to reference it when sending page views to Google Analytics. It should be in the format `UA-XXXXXXXXX-X`.
+
+You can find this tracking ID later by going to `Admin > Tracking Info > Tracking Code`.
+
+Now to configure Gatsby to send page views to your Google Analytics account.
+
+We are going to use `gatsby-plugin-google-analytics`
+
+```npm install --save gatsby-plugin-google-analytics```
```
[the remark this answers is not in the document]
Comment (baumant) [reply docs/docs/adding-analytics.md]: fixed

File: docs/docs/adding-analytics.md [hunk 5]
```diff
@@ -2,7 +2,59 @@
...
+
+Once you have an account you will be prompted to set up a new property. This property will have a Tracking ID associated with it. In this case the property will be the website itself. Fill out the below form with your website name and URL
+
+Once complete you should now have a Tracking ID take note of it as your website will need to reference it when sending page views to Google Analytics. It should be in the format `UA-XXXXXXXXX-X`.
+
+You can find this tracking ID later by going to `Admin > Tracking Info > Tracking Code`.
+
+Now to configure Gatsby to send page views to your Google Analytics account.
+
+We are going to use `gatsby-plugin-google-analytics`, for other analytics options (includeing Google Analytics gtag.js ang Google Tag Manager) check [Other Gatsby analytics plugins](#other-plugins)
```
Comment (DSchau) [inline docs/docs/adding-analytics.md]:
```suggestion
We are going to use `gatsby-plugin-google-analytics`, for other analytics options (including Google Analytics gtag.js ang Google Tag Manager) check [other Gatsby analytics plugins](#other-gatsby-analytics-plugins)
```

Comment (baumant) [reply docs/docs/adding-analytics.md]: fixed

File: docs/docs/adding-analytics.md [hunk 6]
```diff
@@ -2,7 +2,59 @@
...
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        trackingId: "UA-XXXXXXXXX-X",
+      },
+    },
+```
+
+Here replace `UA-XXXXXXXXX-X` with your own Tracking ID.
+
+Full documentation for the plugin can be found in the Gatsby plugins documentation [here](/packages/gatsby-plugin-google-analytics/) or on the Github page [here](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-google-analytics)
```
[the remark this answers is not in the document]
Comment (baumant) [reply docs/docs/adding-analytics.md]: fixed

File: docs/docs/adding-analytics.md [hunk 7]
```diff
@@ -2,7 +2,59 @@
...
+
+There are a number of extra configuration options both with the Gatsby plugin and also on Google Analytics itself so you can tailor to meet your website's needs.
+
+Once this is configured you can deploy your site to test! If you navigate to Home on Google Analytics you should see a dashboard with different statistics.
+
+### <a name="other-plugins"></a>Other Gatsby analytics plugins
```
[the remark this answers is not in the document]
Comment (baumant) [reply docs/docs/adding-analytics.md]: fixed

**Comment by baumant** [comment]
@DSchau any other changes needed?

**Pull request merged**